docs/source/docs/additional-resources/config.md

```diff
@@ -12,20 +12,20 @@ The directory structure is outlined below.
 ```
 
 - calibImgs
-    - Images saved from the last run of the calibration routine
+  - Images saved from the last run of the calibration routine
 - cameras
-    - Contains a subfolder for each camera. This folder contains the following files:
-        - pipelines folder, which contains a {code}`json` file for each user-created pipeline.
-        - config.json, which contains all camera-specific configuration. This includes FOV, pitch, current pipeline index, and calibration data
-        - drivermode.json, which contains settings for the driver mode pipeline
+  - Contains a subfolder for each camera. This folder contains the following files:
+    - pipelines folder, which contains a {code}`json` file for each user-created pipeline.
+    - config.json, which contains all camera-specific configuration. This includes FOV, pitch, current pipeline index, and calibration data
+    - drivermode.json, which contains settings for the driver mode pipeline
 - imgSaves
-    - Contains images saved with the input/output save commands.
+  - Contains images saved with the input/output save commands.
 - logs
-    - Contains timestamped logs in the format {code}`photonvision-YYYY-MM-D_HH-MM-SS.log`. Note that on Pi or Gloworm these timestamps will likely be significantly behind the real time.
+  - Contains timestamped logs in the format {code}`photonvision-YYYY-MM-D_HH-MM-SS.log`. Note that on Pi these timestamps will likely be significantly behind the real time.
```

Comment:
is this note relevant for specifically pi or just in general this would probably be true right

Reply:
@mcm001 is it not possible for us to get timestamps from the fms? at least for matches?

Reply:
That’s not a me question, sorry. The system clock on the Rio will -eventually- sync with the DS time at DS connect, you would need additional photonlib logic to shove that over to all your coprocessors though. Not sure if FMS provides that info

Reply:
The Rio updates logs using the timestamp from the FMS, but I don’t know how we would get this info on the coprocessor or use it to update system time. Maybe we could log something from NT that has this info?

Reply:
we added something like match number right? does that already give us the timestamp?

Reply:
I would assume we could get the current time from the rio and push it to the coprocessor just for logging at least. Out of this prs scope, I updated the wording.

Reply:
I'll cut an issue so we don't forget, but yep, would need a new feature.